mne/coreg.py

```diff
@@ -77,9 +77,8 @@ def create_default_subject(mne_root=None, fs_home=None, update=False,
 
     Parameters
     ----------
-    mne_root : None | str
-        The mne root directory (only needed if MNE_ROOT is not specified as
-        environment variable).
+    mne_root : None
+        This argument is not used anymore.
```

Comment:
I'm wondering if we might be able to offer fsaverage as downloadable dataset, which would change the function signature again and in which case it would be easier to change both arguments at the same time, or whether the signature minus `mne_root` will be stable?